frontend/src/components/bs/components/spaces-table/helpers/cells/delete-row-cell/delete-row-cell.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from './styles.module.scss';
+import deleteIcon from 'assets/img/delete-icon.svg';
+
+const DeleteRowCell = (
+  id: string,
+  onSpaceDelete: (id: string) => void,
+): JSX.Element => {
+  return (
+    <button
+      className={styles.button}
+      onClick={(): void => {
+        onSpaceDelete(id);
+      }}
```

Comment:
```suggestion
  const handleSpaceDelete = () => {
    onSpaceDelete(id);
  }
      onClick={handleSpaceDelete}
```
avoid anonymous functions